test/unit/specs/components/staking/TabMyDelegations.spec.js

```diff
@@ -1,181 +1,246 @@
-import setup from "../../../helpers/vuex-setup"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
+import { shallowMount } from "@vue/test-utils"
 import TabMyDelegations from "renderer/components/staking/TabMyDelegations"
 
-const delegates = lcdClientMock.candidates
+const delegator_addr = `cosmos15ky9du8a2wlstz6fpx3p4mqpjyrm5ctpesxxn9`
+const validators = [
```

Comment:
JSON

Reply:
yeah we are polluting every spec file with those blobs

Reply:
:/ https://github.com/cosmos/voyager/pull/2081#discussion_r260249164

Reply:
done